Resources/Prototypes/Entities/Structures/Piping/Disposal/units.yml

```diff
@@ -85,3 +85,57 @@
     interfaces:
     - key: enum.DisposalUnitUiKey.Key
       type: DisposalUnitBoundUserInterface
+
+
+- type: entity
+  id: MailingUnit
+  parent: DisposalUnitBase
+  name: mailing unit
+  description: A pneumatic mail delivery unit.
+  components:
+  - type: Sprite
+    netsync: false
+    sprite: Structures/Piping/disposal.rsi
+    layers:
+      - state: conmailing
+        map: [ "enum.DisposalUnitVisualLayers.Base" ]
+      - state: mailover-handle
+        map: [ "enum.DisposalUnitVisualLayers.Handle" ]
+      - state: dispover-ready
+        map: [ "enum.DisposalUnitVisualLayers.Light" ]
+  - type: Construction
+    graph: DisposalMachine
+    node: disposal_unit
+  - type: DisposalUnit
+    flushTime: 2
+    autoEngageEnabled: false
+    mobsCanEnter: false
+  - type: MailingUnit
+  - type: DeviceNetwork
+    deviceNetId: Wired
+    receiveFrequencyId: MailingUnit
+    transmitFrequencyId: MailingUnit
+  - type: WiredNetworkConnection
+  - type: Configuration
+    config:
+      tag:
```

Comment:
is it intended that this is empty?

Reply:
It is. Config is a list of configuration values that the player can change using the configuration window and their value in yml is the default value.
I will rewrite the configuration window system in the future.